demonstrations/tutorial_bluequbit.py

```diff
@@ -0,0 +1,172 @@
+r"""Intro to the BlueQubit (CPU) device 
+=============================================================
+
+Here we will show how you can run Pennylane circuits of up to 33 qubits for free on BlueQubit backend.
+Running large scale simulations usually requires lots of memory and compute power.
+Regular laptops already struggle above 20 qubits and most of the time 30+ qubits are a no go.
+Using the BlueQubit device, now Pennylane users can run circuit simulations on souped up machines and go up to 33 qubits!
+
+We use a custom build of Pennilane-Lighning to have multi-threading and other configurations to achieve best possible performance.
+
+.. note::
+
+    To follow along with this tutorial on your own computer, you will need the
+    `BlueQubit SDK <https://app.bluequbit.io/sdk-docs/index.html>`_. It can be installed via pip:
+
+    .. code-block:: bash
+
+        pip install bluequbit
+
+
+.. figure:: ../_static/demonstration_assets/qft/socialthumbnail_large_QFT_2024-04-04.png
+    :align: center
+    :width: 60%
+    :target: javascript:void(0)
+
+
+
+Build your Pennylane circuits as always
+---------------------------------------
+
+Here we will build a simple Bell Pair and simulate it on BlueQubit backend.
+Later in the tuturial we will show a larger example - a 26 qubit quantum adder.
+
+Here is the example circuit we will be simulating:
+"""
+
+import pennylane as qml
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def bell_pair():
+    qml.Hadamard(0)
+    qml.CNOT(wires=(0, 1))
+    return qml.probs()
+
+fig = qml.draw_mpl(bell_pair)()
+
+
+##############################################################################
+# Add the BlueQubit device
+# ---------------------------------------
+# Here are the 3 easy steps to simulate the above circuit on the BlueQubit backend:
+#
+# 1. Open an account at `app.bluequbit.io <https://app.bluequbit.io/>`__ to get a token. You can also view your submitted jobs here later.
+# 2. Initialize the bluequbit device with your token.
+# 3. Submit the circuit for similation!
+
+import bluequbit
+bluequbit.logger.setLevel("ERROR")
+
+# STEP 2: Initialize the bq device!
+# Using a guest token here. Replace it with your own token for better experience.
+bq_dev = qml.device("bluequbit.cpu", wires=2, token="3hmIGLWGKzKdWmxLoJ5F24P3rivGL04d")
+
+bell_qnode = qml.QNode(bell_pair, bq_dev)
+
+# STEP 3: Simulate the circuit!
+result = bell_qnode()
+print(result)
```

Comment:
```suggestion
print(bell_qnode)
```
Hi @hthayko , is this what you were trying to do? Could you please fix this call, make sure that the checks are successful, and then tag me for review?

I've already updated the branch for you and fixed the Poetry discrepancies.

Reply:
Hi @ikurecic - I am printing the result of the simulation here, I believe the code is correct.

If this code failed for you - it might be because of new Pennylane versions. Our sdk currently supports 0.35 and 0.36, but not 0.37. Did you try this code with 0.37 and got an error?

Reply:
Hi @ikurecic - we upgraded our backend to support 0.37 and looks like all checks are passing now!

Regarding the token in the code - that is intentional, I think it's fine - see more [reasoning in my comment above](https://github.com/PennyLaneAI/qml/pull/1127#discussion_r1666775801).

Let me know what else is needed!